deptofdefense/move.mil
# Remove sidenav

## Checklist

I have…


## Summary of Changes

This pull request removes the sidenav from all pages.

Issue #120

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
2. git checkout remove_sidenav
3. set up development dependencies according to CONTRIBUTING.md,
5. run bin/rails server, and
6. load up http://localhost:3000/, check all pages for layout. Add inputs to entitlements and locator maps pages and check new content. Also check all pages with narrow screen.

## Screenshots
<img width="1679" alt="screen shot 2017-10-25 at 2 14 17 pm" src="https://user-images.githubusercontent.com/29409348/32015395-fe1a77fc-b98e-11e7-96b2-0ff0ddf29cd2.png">

Narrow screen:
<img width="397" alt="screen shot 2017-10-25 at 2 17 47 pm" src="https://user-images.githubusercontent.com/29409348/32015520-548a0652-b98f-11e7-99ce-f045db432b50.png">

Service-specific info is the only page whose layout is affected. This is already a problem for certain screen sizes (when the side nav is on the bottom of the page). This should be fixed by this PR though https://github.com/deptofdefense/move.mil/pull/98:
<img width="1675" alt="screen shot 2017-10-25 at 2 14 44 pm" src="https://user-images.githubusercontent.com/29409348/32015478-3499c85a-b98f-11e7-9e17-31ea6a812831.png">
